### PR TITLE
Small fixes following Metabase test

### PR DESCRIPTION
**Pull request by aymeric-dispa**

### CODE REVIEW

File: src/main/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatement.java [hunk 1]
```diff
@@ -241,6 +241,11 @@ public int[] executeBatch() throws SQLException {
 		return result;
 	}
 
+	@Override
+	public int executeUpdate(String sql) throws SQLException {
+		throw new FireboltException("Connect call method executeUpdate(String sql) on a PreparedStatement");
```
Comment (ptiurin) [inline src/main/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatement.java:246]: it's a typo, right? `Cannot`?

File: src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java [hunk 2]
```diff
@@ -140,7 +140,7 @@ public ResultSet getColumns(String catalog, String schemaPattern, String tableNa
 						String.valueOf(columnInfo.getPrecision()), null, // buffer length (not used, see Javadoc)
 						String.valueOf(columnInfo.getScale()), String.valueOf(COMMON_RADIX), // radix
 						columnDescription.getInt("is_nullable") == 1 ? columnNullable : columnNoNulls,
-						columnInfo.getColumnName(), // description of the column
+						null, // description of the column
```
Comment (ptiurin) [inline src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java:143]: What's this change about?

File: src/main/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatement.java (hunk 1 above)
Comment (aymeric-dispa) [reply src/main/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatement.java]: Whoops - just pushed

File: src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java (hunk 2 above)
Comment (aymeric-dispa) [reply src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java:143]: I made a mistake (I set the description of the column as the column name), so now instead I'm setting it to null

**Review by ptiurin — APPROVED** [review]
lgtm, to my knowledge

**Pull request merged**